docs/logs/getting-started-aspnetcore/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "DetailedErrors": true,
```

Comment:
what does this do?

Reply:
I think this is the default experience for development (e.g. showing callstacks).